tests/data_context/store/test_store_backends.py

```diff
@@ -1556,25 +1560,31 @@ def test_InlineStoreBackend(empty_data_context: DataContext) -> None:
     new_config_version: float = 5.0
 
     # test invalid .set
+    key = DataContextVariableKey(resource_type="my_fake_variable")
+    tuple_ = key.to_tuple()
     with pytest.raises(TypeError) as e:
-        inline_store_backend.set(("my_fake_variable", ""), "a_random_string_value")
+        inline_store_backend.set(tuple_, "a_random_string_value")
 
     assert (
-        "Keys in InlineStoreBackend must adhere to the schema defined by DataContextConfig"
+        "Keys in InlineStoreBackend must adhere to the schema defined by DataContextVariableSchema"
         in str(e.value)
     )
 
     # test valid .set
+    key = DataContextVariableKey(resource_type=DataContextVariableSchema.CONFIG_VERSION)
+    tuple_ = key.to_tuple()
     with patch(
         "great_expectations.data_context.DataContext._save_project_config"
     ) as mock_save:
-        inline_store_backend.set(("config_version",), new_config_version)
+        inline_store_backend.set(tuple_, new_config_version)
 
     assert empty_data_context.config.config_version == new_config_version
     assert mock_save.call_count == 1
 
     # test .get
-    ret = inline_store_backend.get(("config_version",))
+    key = DataContextVariableKey(resource_type=DataContextVariableSchema.CONFIG_VERSION)
+    tuple_ = key.to_tuple()
+    ret = inline_store_backend.get(tuple_)
     assert ret == new_config_version
```

Comment:
This test is great - but I do think we also need an integration test to just make sure we read/write to disk (using a mocked filesystem e.g. `pyfakefs`, can be in a separate PR).

Reply:
done!